smac/tae/execute_ta_run.py

```diff
@@ -149,6 +149,12 @@ def start(self, config:Configuration,
                                                           seed=seed,
                                                           instance_specific=instance_specific)
 
+        # Catch NaN or inf.
+        if (self.run_obj == 'runtime' and not np.isfinite(runtime) or
+            self.run_obj == 'quality' and not np.isfinite(cost)):
+            status = StatusType.CRASHED
+            runtime, cost = 0, 0
```

Comment:
I removed the line. Crashed runs won't be considered anyway, so they might aswell have NaNs as cost/runtime.